src/Tribe/Assets.php

```diff
@@ -90,6 +90,7 @@ public function register_in_wp( $assets = null ) {
 	 */
 	public function enqueue( $forcibly_enqueue = null ) {
 		$forcibly_enqueue = (array) $forcibly_enqueue;
+		$localized = array();
```

Comment:
Nice, though I'd suggest making this a class property so it can be populated/checked across multiple calls to `tribe_asset(s)()`.